eunkyung/app.js

```diff
@@ -34,25 +34,144 @@ app.get('/ping', (req, res) => {
 })
 
 app.post('/sign_up', async (req, res, next) => {
-    const { name, password, email } = req.body
+    const { name, password, email, profileImage } = req.body
     await mysqlDataSource.query(
         `INSERT INTO users (
             name, 
             password, 
-            email
+            email,
+            profile_image
             ) 
             VALUES (
+                ?,
                 ?,
                 ?,
                 ?
                 );`,
-        [name, password, email]
+        [name, password, email, profileImage]
     )
     res.status(201).json({ message: 'userCreated' })
 })
 
 
 
+app.post('/posting', async (req, res) => {
+    const { title, content, postImageUrl, userId } = req.body
+    await mysqlDataSource.query(
+        `INSERT INTO posts (
+            title,
+            content,
+            post_image_url,
+            user_id
+        )
+        VALUES (
+            ?,
+            ?,
+            ?,
+            ?
+        );`,
+        [title, content, postImageUrl, userId]
+    )
+    res.status(201).json({ message: 'postCreated' })
+})
+
+app.get('/posts', async (req, res) => {
+    await mysqlDataSource.query(
+        `SELECT
+            users.id as userId,
+            users.profile_image as userprofileImage,
+            posts.user_id as postingId,
+            posts.post_image_url as postingImageUrl,
+            posts.content as postingContent 
+        FROM users 
+        INNER JOIN posts ON users.id = posts.user_id`
+        , (err, rows) => {
+            res.status(201).json({ data: rows })
+        }
+    )
+})
+
+
+app.get('/posts/user', async (req, res) => {
```

Comment:
제가 못봤던 부분인데 엔드포인트 마지막 user 부분은 빼는 것이 좋아보입니다.

우선 머지할테니 참고만 해주세요!

```suggestion
app.get('/posts', async (req, res) => {
```